Update the language for the eligibility section

pixelwake opened the pull request:
@fureigh Created this PR for the change that Cristina requested in Slack.

Commit:
Update the language for the eligibility section

fureigh committed:
Reinstate changes from 5cc2e3d

fureigh commented:
Looks good to me. I made a couple of changes to match what we had before; @rooneywp or @cbrydges, want to review and merge?

Commit:
Update apply.html

Removed "has been admitted to the U.S. for permanent residence" per Cristina's request.

pixelwake commented:
@fureigh I'll merge. I removed "has been admitted to the U.S. for permanent residence" per Cristina's request.
